msteams-platform/graph-api/activity-feed/feed-notifications.md

```diff
@@ -0,0 +1,134 @@
+|sendActivityNotification| Azure Active Directory **userId**| Send a notification to the activity feed of a user. |
+||Sample request ->| **POST** ht<span>tps://graph.microsoft.com/beta/users/{userId}/teamwork/sendActivityNotification |
+|sendActivityNotification| **chatId** | Send a notification to the activity feed of one or more users in a chat. |
+||Sample request ->| **POST** ht<span>tps://graph.microsoft.com/beta/chats/{chatId}/sendActivityNotification
+|sendActivityNotification| **teamId** | Send a notification to the activity feed of one or more users in a team. |
+||Sample request ->| **POST** ht<span>tps://graph.microsoft.com/beta/teams/{teamId}/sendActivityNotification
+
+## Enabling activity feed notifications
+
+The steps for enabling activity feed notifications in your application are as follows:
+
+1. [Register your app with Microsoft identity platform via the Azure AD portal](#register-your-app-with-microsoft-identity-platform-via-the-azure-ad-portal).
+1. [Obtain an access token from the Microsoft Identity platform](#obtain-an-access-token-from-the-microsoft-identity-platform).
+1. [Update your Teams app manifest](#update-your-teams-app-manifest).
+
+## Register your app with Microsoft identity platform via the Azure AD portal
+
+The Azure Active Directory (Azure AD) portal provides a central platform for you to register and configure your apps. Your app must be registered in the Azure AD portal to integrate with the Microsoft identity platform and call Graph APIs. *See* [Register an application with the Microsoft identity platform](/graph/auth-register-app-v2).
+
+>[!WARNING]
+>Do not register multiple Teams apps to the same Azure AD app id. The app id must be unique for each app.
+
+## Obtain an access token from the Microsoft identity platform
+
+To make Microsoft Graph API calls, you must obtain an access token for your app from the Microsoft identity platform. Before your app can get a token from the identity platform, it must be [registered in the Azure AD portal](#register-your-app-with-microsoft-identity-platform-via-the-azure-ad-portal). The access token contains information about your app and the permissions it has for the resources and APIs available through Microsoft Graph.
+
+You'll need to have the following values from the Azure AD registration process to retrieve an access token from the identity platform:
+
+- The **Application ID** assigned by the app registration portal. If your app supports single sign-on (SSO) you should use the same Application ID for your app and SSO.
+- The  **Client secret/password** or a public/private key pair (**Certificate**). This is not required for native apps.
+- A **Redirect URI** (or reply URL) for your app to receive responses from Azure AD.
+
+ *See* [Get access on behalf of a user](/graph/auth-v2-user?view=graph-rest-1.0&preserve-view=true #3-get-a-token) and [Get access without a user](/graph/auth-v2-service).
```

Comment:
Is there a typo in the [Get access on behalf of a user] text? In the preview it doesn't look quite right.

Reply:
Yes, `&preserve-view=true` belongs at the end of the URL`

Reply:
![image](https://user-images.githubusercontent.com/59708116/97057538-4be01900-1540-11eb-9484-eaa3932b3fea.png)
So it is supposed to look like this?

Reply:
I assumed it would be a clickable link.